packages/ra-ui-materialui/src/layout/Confirm.tsx

```diff
@@ -57,12 +57,12 @@ const useStyles = makeStyles(
  */
 const Confirm: FC<ConfirmProps> = props => {
     const {
-        isOpen,
+        isOpen = false,
         loading,
         title,
         content,
-        confirm,
-        cancel,
+        confirm = 'ra.action.confirm',
+        cancel = 'ra.action.cancel',
```

Comment:
why pass a default value for this one and not for other props?

Reply:
I was using type-script with strictMode = true. I'll change it.